gqlalchemy/query_builder.py

```diff
@@ -469,33 +469,42 @@ def from_(
     def where(self, item: str, operator: str, value: Any) -> "DeclarativeBase":
         """Creates a WHERE statement Cypher partial query."""
         separator = "" if operator == ":" else " "
-        self._query.append(
-            WhereConditionPartialQuery(
+        query = (
+            WhereConditionPartialQuery(WhereConditionConstants.WHERE, separator.join([item, operator, value]))
+            if operator == ":"
+            else WhereConditionPartialQuery(
                 WhereConditionConstants.WHERE, separator.join([item, operator, to_cypher_value(value)])
             )
         )
+        self._query.append(query)
 
         return self
 
     def and_where(self, item: str, operator: str, value: Any) -> "DeclarativeBase":
         """Creates a AND (expression) statement Cypher partial query."""
         separator = "" if operator == ":" else " "
```

Comment:
Then we don't need this line, maybe better to put "".join(...) and " ".join(...) in the code?